dist/angular-apimock.js

```diff
@@ -25,18 +25,18 @@ angular.module('apiMock', [])
 		 *
 		 * Public interface:
 		 * `onRequest` method: takes a `request` object and decides if mocking should
-		 *   be done on this request. It checks global and local apiMock flags to see
```

Comment:
No need to change it again, but for future reference: remember to check for unnecessary whitespace changes in commits ^^